app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherViewModel.kt

```diff
@@ -126,6 +126,10 @@ class TabSwitcherViewModel @Inject constructor(
     }
 
     suspend fun purgeDeletableTabs() {
+        Timber.d("TAG_ANA Purging deletable tabs")
```

Comment:
👀

Reply:
It's removed in another PR :see_no_evil: